canonical/juju.is
# updated operator day 2023 and other pages

## Done
- updated operator day 2023 page
- made changes to other operator day pages

## How to QA
- visit https://juju-is-467.demos.haus/operator-day and https://juju-is-467.demos.haus/operator-day/kubecon-eu-2023
- check whether the page matches with [this document](https://docs.google.com/document/d/1HUC1Ja8laGJhByuya8aQ6rsCHUZSwc8uswhaMaqfVp4/edit)

## Issue / Card
Fixes https://warthogs.atlassian.net/browse/WD-3569
